src/mongoChangeStream.test.ts

```diff
@@ -224,6 +224,37 @@ describe('syncing', () => {
     await initialScan.stop()
   })
 
+  test('should emit processError after exhausting retries - initial scan', async () => {
+    const { coll, db } = await getConns()
+    const options: SyncOptions = {
+      retry: {
+        retries: 2,
+        minTimeout: 500,
+      },
+    }
+    const sync = await getSync(options)
+    await initState(sync, db, coll)
+
+    let processError = false
```

Comment:
If we want to be extra explicit about the expected behavior, we could make this a count of processError events and assert that only one occurs.

Reply:
Sounds good.